service/src/test/java/com/google/auto/service/processor/AutoServiceProcessorTest.java

```diff
@@ -38,6 +38,7 @@ public void autoService() {
             JavaFileObjects.forResource("test/SomeServiceProvider1.java"),
             JavaFileObjects.forResource("test/SomeServiceProvider2.java"),
             JavaFileObjects.forResource("test/Enclosing.java"),
+            JavaFileObjects.forResource("test/MultiServiceProvider.java"),
```

Comment:
I think this should be a separate test with just itself and `SomeService`+`AnotherService` to make sure that both get generated. Either that, or we should get the contents and assert that they have what we expect.

Reply:
Good point. Done in d05dca4